feature/container/containerz/tests/container_lifecycle/README.md

```diff
@@ -110,4 +121,10 @@ rpcs:
     containerz.Containerz.StopContainer:
     containerz.Containerz.Log:
     containerz.Containerz.ListContainer:
-```
\ No newline at end of file
+    system.System.Reboot:
+    system.System.SwitchControlProcessor:
+```
+
+## Required DUT Platform
+
+* MFF
```

Comment:
I think you should be able to do this with FFF (unless you're also wanting to check this on a backup supervisor/routing-engine/control-plane in the future).